apis/v1/objectreference.go

```diff
@@ -8,5 +8,6 @@ package v1
 type LocalObjectReference struct {
 	// Name of the referent.
 	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
-	Name string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
+	Name      string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
+	Namespace string `json:"namespace,omitempty" protobuf:"bytes,1,opt,name=namespace"`
```

Comment:
We don't technically need this, but when we keep this we should do it correctly:
```suggestion
	Namespace string `json:"namespace,omitempty" protobuf:"bytes,2,opt,name=namespace"`
```

Reply:
commited it separately